Framework/Algorithms/inc/MantidAlgorithms/SortXAxis2.h

```diff
@@ -0,0 +1,87 @@
+#ifndef MANTID_ALGORITHMS_SORTXAXIS_H_
+#define MANTID_ALGORITHMS_SORTXAXIS_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidAPI/MatrixWorkspace.h"
+#include "MantidAPI/WorkspaceProperty.h"
```

Comment:
It is generally considered good practice to keep the includes to a minimum in the header files.  `System.h` should not be needed at all and this and the 3 below can be moved to the `.cpp` file.